docs/oauth/oauth-single-sign-on.mdx

```diff
@@ -0,0 +1,301 @@
+
+![OAuth scopes openid](/docs/images/oauth/oauth-scopes.png)
+
+You will also need to include this `openid` scope in the query parameters of your authorization endpoint when initiating the OAuth flow. For example:
+
+```bash
+https://clerk.<INSERT_YOUR_APP_DOMAIN>.com/oauth/authorize?client_id=x&response_type=code&scopes=email&state=y&code_challenge=z&redirect_uri=a&scope=openid%20profile%20email&
+```
+
+Once a user successfully authenticates using the OIDC flow, you'll receive:
+
+- An **access token**
+- A **refresh token**
+- An **ID token**
+
+The ID token is a JWT (JSON Web Token) that contains standard JWT claims as defined in RFC 7519, as well as additional custom claims that represent the authenticated user's profile information. The token is signed using your instance's private key.
+
+You must verify the signature on the ID token before using any of the user information it contains. This will require your instance's public key. You can find your instance's public key at `https://clerk.<INSERT_YOUR_APP_DOMAIN>.com/.well-known/jwks.json`.
+
+You can verify the ID token using the `jsonwebtoken` library as an example, like this:
+
+```js
+const jwt = require('jsonwebtoken')
+
+function verifyIdToken(idToken) {
+  jwt.verify(
+    idToken,
+    PUBLIC_KEY,
+    {
+      algorithms: ['RS256'],
+      issuer: 'https://clerk.<YOUR_APP_DOMAIN>.com',
+      audience: '<YOUR_CLIENT_ID>',
+    },
+    (err, decoded) => {
+      if (err) {
+        console.error('Token verification failed:', err)
+      } else {
+        console.log('Token is valid:', decoded)
+      }
+    },
+  )
+}
+```
+
+The ID token includes the following standard claims:
+
+| Standard claim | Description |
+| - | - |
+| `iss` | The issuer of the token, which matches your Clerk Frontend API URL |
+| `sub` | The subject of the token, which matches the authenticated user ID |
+| `aud` | The intended audience of the token, which matches the used Client ID |
+| `exp` | The expiration time of the token |
+| `iat` | The time at which the token was issued |
+| `jti` | A unique identifier for the token |
+
+Depending on the granted scopes, the ID token can include the following additional claims:
+
+| Additional claim | Description |
+| - | - |
+| `nonce` | The nonce provided and used during the request |
+| `family_name` | The user's last name |
+| `given_name` | The user's first name |
+| `name` | The user's full name |
+| `picture` | The user's avatar URL |
+| `preferred_username` | The user's username |
+| `email` | The user's primary email address |
+| `email_verified` | Whether the user's primary email address is verified |
+| `public_metadata` | The user's public metadata |
+| `private_metadata` | The user's private metadata |
+| `unsafe_metadata` | The user's unsafe metadata |
+
+### OIDC prompt
```

Comment:
I don't think this is properly implemented at the moment and has some weird interactions with dynamic client registration and dashboard toggles. I would like to add this back in the future when we look into it, but for now probably should cut